e2e/specs/planner.e2e-spec.ts

```diff
@@ -0,0 +1,152 @@
+                    JudgesCodes[Judges.AMY_WESSOME]),
+                SessionHelper.generateSessionCreateJson(today.clone().minute(15), SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY],
+                    JudgesCodes[Judges.AMY_WESSOME], RoomCodes[Rooms.COURT_4]),
+                SessionHelper.generateSessionCreateJson(today.clone().minute(15), SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY],
+                    JudgesCodes[Judges.JUDGE_LINDA])
+            ];
+    });
+
+    describe('Check if newly created sessions are on planner ', () => {
+        it('Create sessions ', async () => {
+            numberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            expect(numberOfVisibleEvents).not.toEqual(0);
+            for (let session of sessionsToCreate) {
+                expect(await API.createSession(session)).toEqual(200);
+                Logger.log('Created session: id: ' + session.id + ' start: ' + session.start.toISOString());
+            }
+        });
+
+        it('Refresh the view ', async () => {
+            await plannerPage.clickNextButton();
+            await plannerPage.clickPrevButton();
+        });
+
+        it('Refreshed view should show new sessions ', async () => {
+            let newNumberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            visibleElementsDiff = Math.abs(newNumberOfVisibleEvents - numberOfVisibleEvents);
+            Logger.log('Sessions number, before call: ' +
+                numberOfVisibleEvents + ' after: ' + newNumberOfVisibleEvents +
+                ' diff: ' + visibleElementsDiff);
+
+            expect(visibleElementsDiff).toEqual(sessionsToCreate.length);
+        });
+
+        it('There should be at least one not allocated Session', async () => {
+            resourceId = await plannerPage.getResourceIdByName('Not allocated');
+
+            shouldContain = ['No Room', 'No Judge', SessionTypes.FTRACK_TRIAL_ONLY];
+            let found = elementHelper.doesElementContainAllValues(
+                await plannerPage.getSessionEventById(sessionsToCreate[0].id), ...shouldContain);
+            expect(found).toBeTruthy();
+        });
+
+        it('after a click open properly filled dialog', async () => {
+            let event = await plannerPage.getSessionEventById(sessionsToCreate[0].id);
+            const valuesToCheck: string[] = [
+                today.format('DD/MM/YYYY'),
+                SessionTypes.FTRACK_TRIAL_ONLY,
+                '(No judge)',
+                '(No room)'
+            ];
+            await plannerPage.clickAndValidateDialogContent(event, valuesToCheck);
+        });
+
+        it('There should be at least one Session allocated to First judge', async () => {
+            resourceId = await plannerPage.getResourceIdByName(Judges.JUDGE_LINDA);
+
+            shouldContain = ['No Room', Judges.JUDGE_LINDA, SessionTypes.FTRACK_TRIAL_ONLY];
```

Comment:
This ```shouldContain -> found -> expect(found).toBeTruthy()``` is repeating - could we extract this to a method?

Reply:
WET vs DRY, I would stay wet here to allow easy change of each case